pkg/action/cl001/ac006/executor.go

```diff
@@ -77,15 +77,22 @@ func (e *Executor) execute(ctx context.Context) error {
 	var currentMasterPodsHostNetwork int
 	var masterPods []string
 	for _, pod := range masterPodList.Items {
+		// this cronjob pod currently counts against the pods with hostnetwork on master nodes
+		// it will be dropped with the fix for aws-cni v1.7.0, see issue https://github.com/giantswarm/giantswarm/issues/11077
+		if strings.Contains(pod.Name, "aws-cni-restarter") {
+			continue
+		}
 		if pod.Spec.HostNetwork {
 			currentMasterPodsHostNetwork++
 			masterPods = append(masterPods, pod.Name)
 		}
 	}
 
+	expectedPods := "aws-node, calico-node, cert-exporter, k8s-api-healthz, k8s-api-server, k8s-controller-manager, k8s-scheduler, kube-proxy, node-exporter"
```

Comment:
I added all pods which are expected on master and worker so it's easier to see which pod is missing or needs to be added.

Reply:
Would be nice to get a const or var above with some decent godoc so that that magic string is not just buried in the depths of the code.